### ISSUE TITLE
Proposal for low configration users.

### ISSUE THREAD

**Issue by GitHubkaede** [rocksdanister/lively#1788]
- I hope to have an elastic gap when the action or stop is triggered, and it can be a few milliseconds slower when the action or stop is triggered (or customized).
- There is an absolute blacklist, for example, if the XXX program is running, it will completely ignore the list of rules and will not act.
- Set the boot to start, when logging into the Windows system, it will be slower to start moving the wallpaper.
- You can increase the wallpaper playlist by yourself.

**Comment by rocksdanister** [comment]
The translation is not perfect 😅

>I hope to have an elastic gap when the action or stop is triggered, and it can be a few milliseconds slower when the action or stop is triggered (or customized).

Do you mean the wallpaper pausing and unpausing when running fullscreen apps should have a delay?

>There is an absolute blacklist, for example, if the XXX program is running, it will completely ignore the list of rules and will not act.

![image](https://github.com/rocksdanister/lively/assets/17554161/e82ab859-7f80-4c4b-80f5-7a12e979b420)

Is this what you are looking for?

>Set the boot to start, when logging into the Windows system, it will be slower to start moving the wallpaper.

There is this project: https://github.com/sylveon/EarlyStart
but I have not tried it because there can be driver issues that can cause system crash.

>You can increase the wallpaper playlist by yourself.

Automatically change the wallpaper after x amount of time? this is planned for future release.

**Comment by GitHubkaede** [comment]
> The translation is not perfect 😅
> 
> > I hope to have an elastic gap when the action or stop is triggered, and it can be a few milliseconds slower when the action or stop is triggered (or customized).
> 
> Do you mean the wallpaper pausing and unpausing when running fullscreen apps should have a delay?
> 

Yes, because I feel that switching between programs in a short period of time will degrade performance.

> > There is an absolute blacklist, for example, if the XXX program is running, it will completely ignore the list of rules and will not act.
> 
> ![image](https://user-images.githubusercontent.com/17554161/252872400-e82ab859-7f80-4c4b-80f5-7a12e979b420.png)
> 
> Is this what you are looking for?

I know this rule list function, but what I am referring to is the real blacklist, "When a heavy program is executed, although switching it will stop, but the reading between switches will still cause lively to burst in the gap Possibility to run again", (for example: the gap refers to explorer.exe Most of our users will not include explorer.exe in the specification list)
Since I don’t know English and can only use translation, there will be a big gap in explanation. Let me give another example of a situation: I have opened a virtual machine/Adobe Photoshop/large games/various GPU-occupied programs at the same time, but I added "Adobe Photoshop" to the After the blacklist, as long as Adobe Photoshop is executing in the background, the lively desktop will never run.

---------------

In fact, it is not very important, you can refer to it.Thanks.😺
